plugins/multiforward.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: multiforward
+spec:
+  version: v0.0.1
+  homepage: https://github.com/njnygaard/kubectl-multiforward
+  shortDescription: Port Forward to multiple Kubernetes Services
+  description: |
+    This plugin allows you to forward to multiple Kubernetes Services
+    without the use of multiple terminal windows.
+  caveats: |
+    You should have ~/.multiforward.yaml or ./.multiforward.yaml configured.
+    See `multiforward --help` for format details.
+  platforms:
+  - uri: https://github.com/njnygaard/kubectl-multiforward/releases/download/v0.0.3/kubectl-multiforward-linux-amd64.tar.gz
+    sha256: b2d15dc038eac3747f8e388e898529e9f1d7427d1412bf5c4b0a92beb9c1f718
+    bin: kubectl-multiforward
```

Comment:
CI is failing because your archives contain files in
`./artifacts/macos/arm64/kubectl-multiforward`. I'd recommend to flatten the archives, or at least copy the files from the deep folder to a toplevel one (e.g. like here: https://github.com/kubernetes-sigs/krew-index/blob/22ea1e929937bc5546d9e0f12c96de1520e9fc9e/plugins/sshd.yaml#L12-L17).

In addition, you need to include a `LICENSE` file in your dist.

Reply:
Thanks for debugging that for me @corneliusweig! I've flattened the archives and added the `LICENSE` file. That changed the signatures, so I just went with a new release. The bot removed your approval, so whenever you get the time. Thanks for your help!